labels/en-US.json

```diff
@@ -601,6 +601,14 @@
   "shippingTo": "Items shipping to",
   "shipTo": "Ship To",
   "shortDesc": "Summary",
+  "shopperAtStorenotified": "Store has already been notified.",
+  "shopperClosingNotes": "Thank you for shopping with Retailer {0}",
+  "shopperIntransitConfirmationSMS": "Customer {0} is on there way to pickup order number {1}",
+  "shopperInStoreConfirmation": "<p>Thank you for letting us know you are here.</p>",
+  "shopperItemsQty": "Qty:",
+  "shoperNotificationIAmHere": "I'm Already Here",
+  "shopperNotificationText": "<p>Your order is available and ready for you to pick up. Please click the \"I'm on my way\" button to inform the store you are on your way to pick up your order.</p> <p>Thank you so much for your purchase! We're happy we can help you to get what you're looking for.</p> <br/><br/>",
+  "shopperOnMyWayConfirmationText": "<p>Thank you for confirming. The store has been notified, Please contact the store for additional communication.</p>",
```

Comment:
`The store has been notified, Please contact the store for additional communication.`
should be
`The store has been notified. Please contact the store for additional communication.`
That comma should be a period.